modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackTestImages.java

```diff
@@ -3,10 +3,11 @@
 import org.testcontainers.utility.DockerImageName;
 
 public interface LocalstackTestImages {
-    DockerImageName LOCALSTACK_IMAGE = DockerImageName.parse("localstack/localstack:0.12.8");
+    DockerImageName LOCALSTACK_IMAGE = DockerImageName.parse("localstack/localstack:0.13.2");
     DockerImageName LOCALSTACK_0_7_IMAGE = LOCALSTACK_IMAGE.withTag("0.7.0");
     DockerImageName LOCALSTACK_0_10_IMAGE = LOCALSTACK_IMAGE.withTag("0.10.7");
     DockerImageName LOCALSTACK_0_11_IMAGE = LOCALSTACK_IMAGE.withTag("0.11.3");
     DockerImageName LOCALSTACK_0_12_IMAGE = LOCALSTACK_IMAGE.withTag("0.12.8");
+    DockerImageName LOCALSTACK_0_13_IMAGE = LOCALSTACK_IMAGE.withTag("0.13.2");
```

Comment:
What effect is this change supposed to have? This will not automatically add a new test, this will just be an unused constant.